src/system/applications/item/ancestry-sheet.ts

```diff
@@ -0,0 +1,58 @@
+import { AncestryItem } from '@system/documents/item';
+import { DeepPartial } from '@system/types/utils';
+
+import { BaseItemSheet } from './base';
+
+export class AncestrySheet extends BaseItemSheet {
+    static DEFAULT_OPTIONS = foundry.utils.mergeObject(
+        foundry.utils.deepClone(super.DEFAULT_OPTIONS),
+        {
+            classes: ['cosmere-rpg', 'sheet', 'item', 'ancestry'],
+            position: {
+                width: 600,
+                height: 550,
+            },
+            window: {
+                resizable: true,
+                positioned: true,
+                title: '<i class="fas fa-timeline"></i> ' + this.name,
```

Comment:
Good shout. I saw it in the API options and thought I'd try out some changes then forgot to remove when it didn't work.
I still want to try and address this as when minimised, each of the title bars for the item sheets tends to only be able to fit in the type name rather than the actual object name, which isn't helpful.
But someone will need to dig  bit deeper to see how we can override that base DocumentSheet behaviour then...

Reply:
#73 created for investigation

Reply:
removed current experiment